build.sbt

```diff
@@ -2,10 +2,10 @@ organization := "com.spotify.data"
 name := "gcs-tools"
 version := "0.1.8-SNAPSHOT"
 
-val gcsVersion = "1.6.2-hadoop2"
+val gcsVersion = "1.6.3-hadoop2"
 val hadoopVersion = "2.7.4"
-val avroVersion = "1.8.2"
-val parquetVersion = "1.9.0"
+val avroVersion = "1.9.0"
```

Comment:
let's not bump avro in this PR, parquet up to 1.11.0 uses 1.8.2

avro 1.9 as some breaking changes

Reply:
sure. thanks for the review. will update.